packages/kbn-generate-csv/src/generate_csv.ts

```diff
@@ -121,6 +126,7 @@ export class CsvGenerator {
       params: {
         body: searchBody,
       },
+      maxConcurrentShardRequests,
     };
```

Comment:
Based on other code I could find and the way the types are structured, it looks like `max_concurrent_shard_requests` should be deep one level further:

```typescript
    const searchParams: IEsSearchRequest = {
      params: {
        body: searchBody,
        max_concurrent_shard_requests: maxConcurrentShardRequests,
      },
    };
```

Reply:
awesome! changes in bd31abe